fstream-npm.js

```diff
@@ -201,7 +204,7 @@ Packer.prototype.readRules = function (buf, e) {
   if (this === this.root) {
     this.bundleLinks = this.bundleLinks || {}
     this.bundleLinks[p.name] = this._path
-  }
+  }pa
```

Comment:
Pretty sure the `pa` was included by mistake and will cause Bad Things to happen. Remove that?